src/js/dataobjects.js

```diff
@@ -36,6 +36,8 @@ export class Profile {
                 subcategory = self._fixSubcategory(subcategory)
                 Object.values(subcategory.indicators).forEach(indicator => {
                     self._fixMetadata(indicator)
+                    indicator.chartConfiguration = defaults.checkAny(indicator.chartConfiguration, dv.chartConfiguration)
```

Comment:
this is probably the better spot for this default injection. since we already changing the data here.

Reply:
the idea behind the dataobjects chunk of code (needs to be more extensive), is to parse incoming data and normalise it where needed. Where else would you put it?

Reply:
This is the spot :) keep it here. My comment was meant as an encouragement instead of criticism :) sorry about that.